server/views/partials/layout.njk

```diff
@@ -26,6 +26,7 @@
           crossorigin="anonymous"></script>
   <link href="https://code.jquery.com/ui/1.12.1/themes/ui-lightness/jquery-ui.css" rel="stylesheet" nonce="{{ cspNonce }}" crossorigin>
   <link href="/assets/stylesheets/application.css?{{ version }}" rel="stylesheet"/>
+  <link href="/assets/stylesheets/accessible-autocomplete.min.css" rel="stylesheet"/>
```

Comment:
Considering this is only used on one page for now and the for the foreseeable, I'd be tempted to do the same with this as the JS but put it in `{% block head %}` in `/reports/show.njk`.